### PR TITLE
add scott to contributor / code review lists

### PR DESCRIPTION
**Pull request by danawillow**
If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: .ci/containers/terraform-vcr-tester/check_membership.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ USER=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
   "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/${PR_NUMBER}" | jq -r .user.login)
 
 # Only run tests for safe users
-if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson -e melinath); then
+if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson -e melinath -e scottsuarez); then
 	echo "User is on the list, not skipping."
 else
 	echo "Checking GCP org membership"
```
Comment (ScottSuarez) [inline .ci/containers/terraform-vcr-tester/check_membership.sh:13]: I'm in gcp org so this shouldn't be necessary. I suppose it wouldn't hurt though

File: .ci/containers/terraform-vcr-community/reverse_check_membership.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ USER=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
 
 # This image runs tests for community PRs. This script reverses check_membership.sh to exit without running tests
 # for users for who tests are automatically run.
-if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson -e melinath); then
+if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson -e melinath -e scottsuarez); then
```
Comment (ScottSuarez) [inline .ci/containers/terraform-vcr-community/reverse_check_membership.sh:11]: same here

**Review by ScottSuarez — APPROVED** [review]
signing but adding @c2thorn for final review

File: .ci/containers/terraform-vcr-tester/check_membership.sh (hunk 1 above)
Comment (danawillow) [reply .ci/containers/terraform-vcr-tester/check_membership.sh:13]: @slevenick can we just remove all the googlers from this list? that makes sense, right?

**Review by c2thorn — APPROVED** [review]

File: .ci/containers/terraform-vcr-tester/check_membership.sh (hunk 1 above)
Comment (slevenick) [reply .ci/containers/terraform-vcr-tester/check_membership.sh:13]: Yeah that makes sense. @ScottSuarez have builds been triggering automatically for you?

Comment (ScottSuarez) [reply .ci/containers/terraform-vcr-tester/check_membership.sh:13]: yep ! They've been triggering for me

**Pull request merged**